bool jump
uint8 speed
float64 x
float64 y
float64 z
float64 pitch
float64 yaw